velox/dwio/dwrf/reader/ReaderBase.h

```diff
@@ -247,6 +255,7 @@ class ReaderBase {
   std::unique_ptr<encryption::DecryptionHandler> handler_;
   std::shared_ptr<dwio::common::BufferedInputFactory> bufferedInputFactory_;
 
+  dwio::common::FileFormat fileFormat_;
```

Comment:
nit: since we already have a flat in PostScript class, why not push this to PostScript, and here we can just return whatever is captured in PostScript?

Reply:
Done.